### PR TITLE
Arconomy: Civilian Bounty Redux and full replacement of cargo bounties.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
Gonna wear that name down all the way to a bare spot on the floor.

## About The Pull Request

Alright, this is a functional rework for civilian bounties, which should serve a few purposes:
* Cargo has a reason to actually keep working and stay within cargo.
* Makes cargo bounties far more integrated into the round as opposed to being a static list of soft goals, being personalized to those who would actually do a bounty for the station.
* Still make civilian bounties the prime, sirloin steak method for regular crew to make money.

So here's the 4-11:
Static cargo bounties have been removed, in favor of the new system of civilian bounties. That means that both the bounties app and the bounty console have been removed.

Civilian bounties have been buffed. They are no longer affected by inflation, but when your bounty is completed, you will recieve a "bounty cube". To receive the cash you would have received for the bounty (10% of the civilian bounty), that bounty cube must be shipped off the cargo shuttle, and using the magic of the price tag component, you make your cut, and cargo makes their 90%, ensuring that you'll actually see your money (for once!).

Civilian bounties are now independent of department budgets, in preparation for departmental purchases, so now you can more freely benefit from doing civilian bounties to buy things! (See #53881)

Non-mining cargo department crew now have fully randomized bounties, to give them stationwide goals for bounty hunting.

As an added benefit, the icon for bounty cubes basically screams value, so hopefully crewmembers will see it and instantly think to sell it.

## Why It's Good For The Game

Base cargo bounties are neat, but at the same time, they're skewed towards jobs that aren't even certain to be on the station. You can have Viro bounties, Botany bounties, chemistry bounties, all giving a reward for tasks that aren't even certain to be on station, and as a cargo tech, you need to get other departments to do 90% of the work for you, likely without seeing an ounce of the reward, as cargo has no incentive to reward you for your efforts, (And likely can't anyway, since they can't take the money out to pay you from the cargo budget.

Bam, now civilian bounties benefit both you and the cargo department. Suddenly bounties require the rest of the station to help cargo work for you. Cargo can work on bounties for themselves in addition to the rest of the crew, and let more meaningful money enter the system, while putting larger quantities of money in the hands of crew who are going to the effort to buy things.

## Changelog
:cl:
add: Completing a bounty now rewards a hardlight, bounty cube that must be exported for your payout.
del: Cargo no longer has static, roundstart bounties.
del: In addition, the Bounty App and Bounty Console have been removed.
balance: Civilian bounties have a higher payout now, but require larger quantities of their target object, as before.
balance: Cargo Tech, QM, and HOP bounties are now fully random bounties, as opposed to assistant bounties.
/:cl:

## To-Do

- [x] Fix the maps to no longer include bounty consoles.
- [x] Determine what more work needs to be done to keep the variety and flavor of existing bounty types fresh. (Looking at you, security.)
- [x] Get some testmerge feedback.

### CODE REVIEW

**Comment by Timberpoes** [comment]
I like the direction this is headed in!

**Comment by Winterous** [comment]
I hope it's not possible for cargo to remove your cut of the bounty like they can with package price tags if they're feeling particularly monstrous.
Also will this 10% cut be around what you get for private bounties right now? Because I think the payout needs to be a bit higher than what it is right now.

**Comment by optimumtact** [comment]
goddamn it how many fucking bounty systems are we going to go through before admitting they don't work?

**Comment by Tlaltecuhtli** [comment]
will the civilian bounty gains be increased? (after the nerf people stopped using them) as now cargo will need  10 times more bounties to reach what you would get before

**Comment by ArcaneMusic** [comment]
> will the civilian bounty gains be increased? (after the nerf people stopped using them) as now cargo will need 10 times more bounties to reach what you would get before

Yes they have.
Example:
You as a Medical Doctor get a bounty for 3 Livers or something, reward is 5k.
You collect and turn in the 3 livers, and receive a bounty cube.
You pass the bounty cube off to cargo, who sends it off on their next shipment.
The cargo department receives 4500 credits, where you receive 500 credits, which is in-between the first wave of civilian bounty payments and the current post-nerf values. 
I have tested this against several department's bounties, and there are none thusfar that feel out of place for the amount of required effort and time afforded to the task.

**Comment by Krysonism** [comment]
I like the cargo integration. 

I think the biggest problem is the low quality of many of the bounties themselves though.

**Comment by Tlaltecuhtli** [comment]
can you give cargo department ids access to all bounties instead of assistant only?

**Comment by Jared-Fogle** [comment]
"**civilain**_bounty.dm"

File: code/modules/cargo/exports/civilain_bounty.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/datum/export/bounty_box
+	cost = 1
+	k_elasticity = 0 //Bounties are non-elastic funds.
+	unit_name = "completed bounty cube"
+	export_types = list(/obj/item/bounty_cube)
+
+/datum/export/bounty_box/get_cost(obj/O, allowed_categories, apply_elastic)
+	. = ..()
+	var/obj/item/bounty_cube/cube = O
+	return cube.bounty_value
```
Comment (Jared-Fogle) [inline code/modules/cargo/exports/civilain_bounty.dm:10]:
```suggestion
/datum/export/bounty_box/get_cost(obj/item/bounty_cube/cube, allowed_categories, apply_elastic)
	return cube.bounty_value
```

**Review by Twaticus — APPROVED** [review]

**Comment by TheVekter** [comment]
This is an interesting change. I'm not 100% sure how I feel about it removing cargo's normal bounties (these should stay IMO with public bounties as an option to make more dosh) but I'd be interested in seeing these changes in action either way.

Is the goal to increase the amount that players get from these wholesale? As it is right now, most of the civilian bounties just aren't really worth doing.

**Pull request merged**